pkg/app/server/apikeyverifier/verifier.go

```diff
@@ -28,6 +28,7 @@ import (
 
 type apiKeyGetter interface {
 	Get(ctx context.Context, id string) (*model.APIKey, error)
+	UpdateLastUsedAt(ctx context.Context, id, projectID string) error
```

Comment:
the name of the interface is XXXGetter, so it should contain only getter methods, not update methods. We should update the name of this interface to apiKeyStore.